src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -66,7 +68,9 @@ public PersistentDataStore() {
         String userName = (String) entity.getProperty("username");
         String password = (String) entity.getProperty("password");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        User user = new User(uuid, userName, password, creationTime);
+        Boolean is_admin = (Boolean) entity.getProperty("is_admin");
```

Comment:
Does this method allow you to return primitives? That way, you don't need to do a ternary statement on the next line.

Reply:
Yes it does. When I half implemented all the methods, is_admin threw NullPointerExceptions. But this is no longer an issue so I can use primitive Boolean everywhere now.